src/nextjournal/clerk/builder.clj

```diff
@@ -293,11 +293,7 @@
                                                                             paths/*build-opts* opts
                                                                             viewer/doc-url (partial doc-url opts file)]
                                                                     (let [doc (eval/eval-analyzed-doc doc)]
-                                                                      (assoc doc :viewer (view/doc->viewer (assoc opts
-                                                                                                                  :nav-path (if (instance? java.net.URL file)
-                                                                                                                              (str "'" (:ns doc))
-                                                                                                                              (str file)))
-                                                                                                           doc))))
+                                                                      (assoc doc :viewer (view/doc->viewer (assoc opts :file-path (str file)) doc))))
```

Comment:
I'm unsure whether I should keep also a `:nav-path`, but for a static app it is not used by the router nor by the viewer (e.g. in the header) I think.